### ISSUE TITLE
Unit testing

### ISSUE THREAD

**Issue by anidev** [anidev/frc-driverstation#12]
Unit testing should be implemented for at least the network API, but ideally for as much of the code as possible.

**Comment by LoadingPleaseWait** [comment]
I agree with you 100%. This project could benefit from unit testing. I will try to implement this using junit.

**Comment by LoadingPleaseWait** [comment]
I think a good place to put the unit tests might be java/ProjectName/src/test. Also, for using junit with Apache Maven, we might use the [Maven Surefire plugin](https://maven.apache.org/surefire/maven-surefire-plugin/examples/junit.html).

**Comment by anidev** [comment]
From what I understand, Maven expects unit tests to be in <module>/src/test/java. Some of the individual modules here have that directory and some don't, not sure why. Seems like Surefire is how Maven does unit testing with JUnit, so yeah we should use that.

**Comment by LoadingPleaseWait** [comment]
I edited `pom.xml` in FRCDS-Java-Comm to include junit as a dependency. This will make it so we can now use Surefire. I'll send that in with my next pull request.
